RubeesEat.IntegrationTests/SeleniumTests/SplitBillSeleniumTests.cs

```diff
@@ -9,7 +9,7 @@ public void AddPerson()
     {
         var page = Start<SplitBillPageObject>();
         
-        page.ClickAddPerson();
```

Comment:
With the page object pattern all the finding stuff should be located in the page object and not added in the test. Previously, we called `ClickAddPerson` and the method on the page object decided what to do. Now we supply the action directly, which is not what we want. Please move the `InvokeAction` into the method of the page object.